### DISCUSSION TITLE
How to disable CMD email relay

### DISCUSSION THREAD

**Discussion by nukulyadav** [postalserver/postal#3004] [Help with using Postal]
Hi,

I'm using postal for sometime now, I just found out someone is able to relay email from CMD CLI.

Please guide how to disable or restrict that?

Thanks

**Comment by willpower232** [comment]
Can you describe a bit more about what is happening on the cli?

**Reply by nukulyadav** [reply]
Apologies @willpower232  for my delayed response.

Here are the details.

1. Let's say company A has an Email Server in Postal and they are able to send emails normally.
2. Now if "someone" knows the assigned hostname in postal (ex - mail.domain.com) then they can relay the emails from CLI/CMD Ex. - https://support.managed.com/kb/a2313/how-to-send-email-locally-using-command-prompt.aspx

This open a risk for spoofing and can lead to major issue.

**Reply by willpower232** [reply]
Postal is able to [receive incoming email](https://docs.postalserver.io/welcome/feature-list#incoming-e-mails) and also must have port 25 open so that bounces can be delivered back to it.

I'm not sure what the problem is?

**Reply by nukulyadav** [reply]
@willpower232 - The issue is that any unauthorized person (who knows the Hostname) is able to send using postal server.

Example : Organization ABC has postal and someone outside of organization knows hostname, so that person can send email via CMD.

In that case, the person can send illegit email using ABC company hostname.
